src/mapgen/mg_biome.h

```diff
@@ -63,8 +63,12 @@ class Biome : public ObjDef, public NodeResolver {
 	s16 depth_water_top;
 	s16 depth_riverbed;
 
+	s16 x_min;
+	s16 x_max;
 	s16 y_min;
 	s16 y_max;
+	s16 z_min;
+	s16 z_max;
```

Comment:
Connected with the last comment, these here could also be replaced by two `v3s16` vectors.